address.go

```diff
@@ -64,3 +82,24 @@ func Latitude(opts ...options.OptionFunc) float64 {
 		return float64(address.latitude())
 	}, opts...).(float64)
 }
+
+type RealCoordinates struct {
+	Latitude  float64 `json:"lat"`
+	Longitude float64 `json:"lng"`
+}
+
+type RealAddress struct {
+	Address1    string          `json:"address1"`
+	Address2    string          `json:"address2"`
```

Comment:
What are this address1 and address2 represent to?
Do we have a better naming?

Reply:
`address1` and `address2` are copy from original json file.
looks like `address1` is just normal address, `address2` is empty in mostly case.
I will remove `address2` just now.